src/workerd/io/compatibility-date.capnp

```diff
@@ -220,4 +220,9 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
   # The Worker can also request specific compression settings by specifying a valid
   # Sec-WebSocket-Extensions header, or setting the header to the empty string to explicitly
   # request that no compression be used.
+
+  nodeCompatExperimental @21 :Bool
+        $compatEnableFlag("node_compat_experimental");
+  # Experimental, do not use.
+  # Enables node compat imports in the application.
```

Comment:
Unfortunately, I think folks are still going to use it regardless of us saying don't use it. Perhaps there is another way we can enable this via some other less permanent mechanism for now? Say, maybe if the `workerd` binary is run with the `--experimental` CLI flag then this can be enabled without needing to introduce a compat flag? What do you think @kentonv ?

Reply:
I wonder if it would be possible for edgeworker to have a separate listing of internal-only compat flags.

Reply:
Compat flags that don't define an enable date automatically require `--experimental` to use with workerd. However, we don't currently have any such experimental guard for using the same flag on Cloudflare Workers. We probably should, or maybe even a way to block access except to approved alpha testers.

There is a comment in this file noting that such flags are subject to change, at least.

Reply:
@kentonv ... if I'm interpreting your comment correctly, you're ok with the approach in this PR? If so, I'll clear my "request changes" and approve.

Reply:
I was just pointing out some facts. That said, I don't see any other way forward here, unless we're going to force Mike to do the work of implementing guards against experimental flags in the deployment service, which seems unfair. So yeah, this seems fine.